examples/iam/rolepolicy.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: identity.aws.crossplane.io/v1alpha1
+kind: IAMRolePolicy
+metadata:
+  name: somerolepolicy
+spec:
+  forProvider:
+    PolicyName: external-name
```

Comment:
I was going to suggest that you consider using external name here, but you've already done it! So I think this line should be deleted from the example.